src/reactviews/pages/QueryResult/commandBar.tsx

```diff
@@ -11,7 +11,12 @@ import * as qr from "../../../sharedInterfaces/queryResult";
 import { locConstants } from "../../common/locConstants";
 import { saveAsCsvIcon, saveAsExcelIcon, saveAsJsonIcon } from "./queryResultUtils";
 import { QueryResultSaveAsTrigger } from "../../../sharedInterfaces/queryResult";
-import { ArrowMaximize16Filled, ArrowMinimize16Filled } from "@fluentui/react-icons";
+import {
+    ArrowMaximize16Filled,
+    ArrowMinimize16Filled,
+    Grid16Regular,
+    Document16Regular,
```

Comment:
Confirm these icons are ones we like.  https://react.fluentui.dev/?path=/docs/icons-catalog--docs

Maybe instead `TableRegular` and `TextColumnThreeRegular`